### PR TITLE
Blips collected from:  chamber cardio, capable health, auxa health, attend, diana health

### PR DESCRIPTION
**Pull request by jorge-vasquez-acn**
Automatically generated by Netlify CMS

### CODE REVIEW

**Review by annaelleduff — APPROVED** [review]

File: src/content/blips/bonfhir.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+description: >-
+  Bonfhir is a collection of open-source building blocks (technically NPM packages) that can be used to create products and solutions based on the [HL7 FHIR](https://fhir.org/) standard.  The vision of the team behind it is to become the equivalent of [retool](https://retool.com/) for FHIR.  
+opinion: >-
+  It allows you to save a lot of time and reuse boilerplate code (Ex:  Formatters).  A team working on a health-tech project can save up to 6 months of work by leveraging bonfhir.  
+  
+  
+  As of Jul 2023 it is still in beta release and therefore lacking maturity and documentation.
+  
+
+  Also note that onboarding and integrating the framework at this moment still requires some advanced engineering and architecture knowledge.  The team is working on simplifying the adoption and the requiried technical entre level.
```
Comment (lp) [inline src/content/blips/bonfhir.md:12]: _entry_ level

File: src/content/blips/expo.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+link: 
+  - https://expo.dev/
+ring: adopt
+quadrant: platform
+businessModel:
+  - open-source
+  - saas
+projectIds:
+  - attend
```
[the remark this answers is not in the document]
Comment (lp) [reply src/content/blips/expo.md:31]: we used it also at some point in Capable, for the initial POC CapableCare app

File: src/content/blips/figma.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+link: 
+  - https://www.figma.com/
+ring: adopt
+quadrant: tool
+businessModel:
+  - saas
+projectIds:
+  - attend
+  - diana-health
```
[the remark this answers is not in the document]
Comment (lp) [reply src/content/blips/figma.md:22]: if not _all_, Capable & Cumulus were/are using it

File: src/content/blips/maestro.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  Maestro is a mobile UI testing framework that facilitates the testing of user journeys inside an app.   
+opinion: >-
+  It has the following strengths:
+  
+  - Easy to get up and running
+
+  - Very good support (Ex:  They have a dedicated slack channel)
+
+  - Very good documentation
+
```
Comment (lp) [inline src/content/blips/maestro.md:13]: No weakness?

File: src/content/blips/mixpanel.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
+---
+title: mixpanel
+description: >-
+  Mixpanel is primarily a product analytics platform, designed to give marketers and product teams insights into how to best acquire, convert, and retain customers, with real-time data across devices and channels.
+opinion: >-
+  It has the following strengths:
+  
+  - Its event-centric data model that combines user and behavioural data is simple to understand
+
```
Comment (lp) [inline src/content/blips/mixpanel.md:9]: No weakness?

File: src/content/blips/monorepo.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  A monorepo facilites the following:
+
+  - Code Reusability
+
+  - Dependency Management
+  
+  - Easier Refactoring and Code Maintenance
+  
+  - Simplified Testing and Continuous Integration
+
```
Comment (lp) [inline src/content/blips/monorepo.md:15]: No weakness?

File: src/content/blips/sentry.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+link: 
+  - https://sentry.io/
+ring: adopt
+quadrant: platform
+businessModel:
+  - saas
+projectIds:
+  - attend
+  - diana-health
+---
```
Comment (lp) [inline src/content/blips/sentry.md:21]: also used by Capable

File: src/content/blips/shortcut.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  - Some advanced features are hidden and not easy to access and figure out (Ex:  ticket templates)
+
+link: 
+  - https://www.shortcut.com/
+ring: adopt
+quadrant: tool
+businessModel:
+  - saas
+projectIds:
+  - attend
```
Comment (lp) [inline src/content/blips/shortcut.md:27]: also used by Capable & Cumulus

**Review by lp — APPROVED** [review]

File: src/content/blips/maestro.md (hunk 4 above)
Comment (jorge-vasquez-acn) [reply src/content/blips/maestro.md:13]: @annaelleduff:  I didn't collect any weakness during our conversation on this, do you have any?

File: src/content/blips/mixpanel.md (hunk 5 above)
Comment (jorge-vasquez-acn) [reply src/content/blips/mixpanel.md:9]: @annaelleduff: I didn't collect any weakness during our conversation on this, do you have any?

File: src/content/blips/maestro.md (hunk 4 above)
Comment (julienblin) [reply src/content/blips/maestro.md:13]: +1 I believe we need a bit more content there on the caveats or recommendations

File: src/content/blips/mantine.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (julienblin) [inline src/content/blips/mantine.md:1]: Let's put this on the back burner until we have more content.
(Hint: Adrienne is working on a Design system in Figma to mirror what's in Mantine...)

File: src/content/blips/medplum.md [hunk 10]
```diff
@@ -3,20 +3,20 @@ title: Medplum
 description: Medplum is a headless EHR that makes it easier to build healthcare
   apps quickly with less code.
 opinion: >-
-  * H﻿eadless [FHIR](https://hl7.org/fhir) server that allows implementation of
+  * Headless [FHIR](https://hl7.org/fhir) server that allows implementation of
```
Comment (julienblin) [inline src/content/blips/medplum.md:6]: Should we align with this in terms of format with the others (e.g. Strenght/weaknesses)?

File: src/content/blips/medplum.md [hunk 11]
```diff
@@ -3,20 +3,20 @@ title: Medplum
...
+  * Awesome Authentication & Authorization model that:
 
-    * c﻿an map authentication from different external identity providers (Auth0, Google, Clerk...)
-    * m﻿eans that you don't need to create a proxy in front of the server - it can be used directly from the front-end
-  * S﻿mall company, very knowledgeable and responsive
+    * can map authentication from different external identity providers (Auth0, Google, Clerk...)
+    * means that you don't need to create a proxy in front of the server - it can be used directly from the front-end
+  * Small company, very knowledgeable and responsive
 
   * The FHIR spec is not entirely supported, and it is sometime hard to know what works and what does not, particularly around the search
```
Comment (julienblin) [inline src/content/blips/medplum.md:16]: another caveat:
 - support for [Terminology Service](https://www.hl7.org/fhir/terminology-service.html) is very limited and will probably require something else on a complex project.

File: src/content/blips/mixpanel.md (hunk 5 above)
Comment (julienblin) [reply src/content/blips/mixpanel.md:9]: Also, we should mention:
 - they have a startup program - 1 year free: https://mixpanel.com/startups
 - HIPAA compliance: https://mixpanel.com/legal/mixpanel-hipaa

File: src/content/blips/pdfplumber.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
+---
+title: pdfplumber
+description: >-
+  Pdf parser that simplifies the extraction of detailed information about each char, rectangle, line, etc. — and easily extract text and tables.
```
Comment (julienblin) [inline src/content/blips/pdfplumber.md:4]: I would add clearly that this is a Python library.

Maybe some caveats / things to look / limitations as well?

File: src/content/blips/pgvector.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+
+link: 
+  - https://gitee.com/hailei_yan/pdfplumber
+ring: adopt
+quadrant: tools
+businessModel:
+  - open-source
+projectIds:
+  - auxa-health
```
Comment (julienblin) [inline src/content/blips/pgvector.md:24]: Isn't CampFHIR using a vector database? @lp

File: src/content/blips/pgvector.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
+---
+title: pgvector
+description: >-
+  Postgres extension for vector similarity search
+opinion: >-
```
Comment (julienblin) [inline src/content/blips/pgvector.md:5]: We could add that this is supported in supabase: https://supabase.com/docs/guides/database/extensions/pgvector

File: src/content/blips/contentful.md [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+  - Its setup requires a technical person to model the content
+
+link: 
+  - https://www.contentful.com/
+ring: adopt
+quadrant: platforms
+businessModel:
+  - saas
+projectIds:
+  - diana-health
```
[the remark this answers is not in the document]
Comment (julienblin) [reply src/content/blips/contentful.md:21]: + did you end up using at resolve as well, @lvancraen?

File: src/content/blips/launchdarkly.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+  - Getting up and running is fast but things get complex as team sizes scale up
+
+
+link: https://launchdarkly.com/
+ring: trial
+quadrant: tools
+businessModel:
+  - saas
+projectIds:
+  - capable-health
```
[the remark this answers is not in the document]
Comment (julienblin) [reply src/content/blips/launchdarkly.md:32]: Is Attend using LaunchDarkly? @annaelleduff

File: src/content/blips/maestro.md [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+link: 
+  - https://maestro.mobile.dev/
+ring: adopt
```
Comment (julienblin) [inline src/content/blips/maestro.md:16]: Definitely not adopt - either Trial or Assess; I would go for Assess given our limited experience still

File: src/content/blips/mixpanel.md [hunk 18]
```diff
@@ -0,0 +1,18 @@
...
+link: 
+  - https://mixpanel.com/
+ring: adopt
+quadrant: tool
+businessModel:
+  - saas
+projectIds:
+  - attend
```
Comment (julienblin) [inline src/content/blips/mixpanel.md:17]: + ensage

File: src/content/blips/monorepo.md [hunk 19]
```diff
@@ -0,0 +1,24 @@
...
+
+link: https://en.wikipedia.org/wiki/Monorepo
+ring: adopt
```
Comment (julienblin) [inline src/content/blips/monorepo.md:18]: IF we only have 1 project (and some libraries such as BonFHIR), then it should be Trial

File: src/content/blips/pdfplumber.md [hunk 20]
```diff
@@ -0,0 +1,23 @@
...
+  - Supports the parsing of table appearances
+
+  - Transforms pdf documents into cartesian planes
+
+  - It is very memory efficient since it uses generators leveraging python streams
+
+
+link: 
+  - https://gitee.com/hailei_yan/pdfplumber
+ring: adopt
```
Comment (julienblin) [inline src/content/blips/pdfplumber.md:17]: Since it's only on one project, it should be at most Trial

File: src/content/blips/pgvector.md [hunk 21]
```diff
@@ -0,0 +1,25 @@
...
+  It has the following weaknesses:
+
+  - Testing is difficult
+
+  - Fine tuning is difficult
+
+
+link: 
+  - https://gitee.com/hailei_yan/pdfplumber
+ring: adopt
```
Comment (julienblin) [inline src/content/blips/pgvector.md:19]: Is this at the adopt phase, or should we put it to Trial until we hve more usage?

File: src/content/blips/render.md [hunk 22]
```diff
@@ -0,0 +1,33 @@
...
+
+
+link: 
+  - https://render.com/
+ring: assess
+quadrant: platforms
+businessModel:
+  - saas
+projectIds:
+  - capable-health
```
Comment (julienblin) [inline src/content/blips/render.md:32]: I am not sure Capable _really_ used render - @lp would you be able to confirm?

File: src/content/blips/retool.md [hunk 23]
```diff
@@ -0,0 +1,22 @@
...
+  It is worth noting that you pay for every user of the application (both the users that build the apps referred to as:  "Standard Users" and the users that use the apps referred to as "end users" need to pay a monthly fee) so for companies with a lot of employees and potential users of each app it might be worth doing a cost analysis. However, we think that for small companies it definitely pays off.  For companies that require SSO the only option is the Enterprise version where it can get expensive.  
+  
+link: https://retool.com/
+ring: adopt
+quadrant: platforms
+businessModel:
+  - saas
+  - on-premise
+projectIds:
+  - chamber-cardio
```
Comment (julienblin) [inline src/content/blips/retool.md:21]: I believe Rogo used it - @rimthong could confirm.
I also belive at least  another project used it, we should ask in the devs channel probably...

File: src/content/blips/retool.md [hunk 24]
```diff
@@ -0,0 +1,22 @@
+---
+title: retool
+description: >-
+  Retool is a low-code application platform that makes it easy to build custom apps, dashboards, admin panels, and other internal tools to run critical business operations.  
+opinion: >-
+  We believe that low-code tools dramatically accelerate the customization of business workflows and retool excels at this by allowing slightly tech users to have a good place to start building front-ends and customize their business workflows.
```
Comment (julienblin) [inline src/content/blips/retool.md:6]: This format is different than others.

File: src/content/blips/sentry.md (hunk 7 above)
Comment (julienblin) [reply src/content/blips/sentry.md:21]: Also used by reside and chamber

File: src/content/blips/sentry.md [hunk 25]
```diff
@@ -0,0 +1,21 @@
...
+title: sentry
+description: >-
+  Sentry is an error-tracking and performance monitoring platform platform that helps developers monitor live applications. 
+opinion: >-
```
Comment (julienblin) [inline src/content/blips/sentry.md:5]: We should be able to add more to this.

File: src/content/blips/shortcut.md [hunk 26]
```diff
@@ -0,0 +1,29 @@
+---
+title: shortcut
+description: >-
+  Shortcut allows teams to collaborate, plan, and visualize work. Users can track and manage tasks through stories, milestones, epics, and iterations.
+opinion: >-
+  It has the following strengths:
```
Comment (julienblin) [inline src/content/blips/shortcut.md:6]: I think we that we should mention the elephant in the room here (JIRA), and explain why this is better.

File: src/content/blips/shortcut.md [hunk 27]
```diff
@@ -0,0 +1,29 @@
...
+  
+  - Its free plan offers a lot of features although some important ones such as reports are not available
+
+  - It is flexible and supports both kanban or Scrum
+
+  - Supprots customizable filters that can be shared
```
Comment (julienblin) [inline src/content/blips/shortcut.md:12]:
```suggestion
  - Supports customizable filters that can be shared
```

File: src/content/blips/swift.md [hunk 28]
```diff
@@ -2,7 +2,7 @@
 title: Swift
```
Comment (julienblin) [inline src/content/blips/swift.md:2]: This should be removed.

File: src/content/blips/zincsearch.md [hunk 29]
```diff
@@ -0,0 +1,34 @@
...
+
+  - No built-in High Availability
+
+  - It is not a cloud managed services and more heavy-lifting is required to operate it
+
+  - etcd is required
+
+link: 
+  - https://github.com/zincsearch/zincsearch
+ring: adopt
```
Comment (julienblin) [inline src/content/blips/zincsearch.md:28]: This should be no higher than Trial, as we only have 1 project usage. Great find though!

File: src/content/projects/rogo.md [hunk 30]
```diff
@@ -7,7 +7,7 @@ description: Rogo empowers your team to uncover data insights – no SQL or
   forward.
```
Comment (julienblin) [inline src/content/projects/rogo.md:7]: I did not see anything from Rogo, although I feel they had quite a run...

File: src/content/projects/synapsebio.md [hunk 31]
```diff
@@ -4,7 +4,7 @@ description: SynapseBio uses Machine Learning and EEG to Improve Detection and
   Treatment of Dementia and Other Neurological and Psychiatric Disorders
```
Comment (julienblin) [inline src/content/projects/synapsebio.md:4]: I did not see anything from Synapse. I feel like they might bring interesting Blips as what they do was quite different from what we usually do.

**Review by julienblin — APPROVED** [review]
As discussed, I will approve this PR in order to not block the process, although I feel like there is a lot of work still.

I would appreciate you at least remove the blips that are clearly not ready, and place them in their own individual PRs for them to be worked on.

File: src/content/blips/contentful.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (annaelleduff) [reply src/content/blips/contentful.md:21]: you're right!

File: src/content/blips/figma.md [hunk 32]
```diff
@@ -0,0 +1,23 @@
+---
```
[the remark this answers is not in the document]
Comment (annaelleduff) [reply src/content/blips/figma.md:1]: was the design team working on a design system for startups? would that qualify figma? if it does then we should probably link to said design system as that would be the differenciator from just figma as a tool itself

File: src/content/blips/launchdarkly.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (annaelleduff) [reply src/content/blips/launchdarkly.md:32]: we aren't, we use Flagsmith because it's free!

[the remark this answers is not in the document]
Comment (julienblin) [reply src/content/blips/launchdarkly.md:32]: So... should that be another Blip? 😉 😉 ???

**Pull request merged**